core/templates/dev/head/css/oppia.css

```diff
@@ -31,6 +31,35 @@
   100% { opacity: 1; }
 }
 
+@keyframes fadein {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+/* Firefox < 16 */
+@-moz-keyframes fadein {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+/* Safari, Chrome and Opera > 12.1 */
```

Comment:
Oh, ignore that. Though I think you mean 15.0? https://www.w3schools.com/cssref/css3_pr_animation-keyframes.asp

Also, I'm wondering how important it is to support such old versions -- I was looking at [this](https://www.netmarketshare.com/browser-market-share.aspx?qprid=2&qpcustomd=0). Would we need to do that elsewhere too? Would appreciate your thoughts.